### ISSUE TITLE
ignore case sort

### ISSUE THREAD

**Issue by grylem** [lille-boy/alphabet_soup#1]
Hello
please add option for case-independent sorting like AaBbCc.
Thanks.

**Comment by lille-boy** [comment]
Hello grylem,
Thanks for the suggestion.
I have a working solution on a branch. Trying to see if I can optimize it.

**Comment by grylem** [comment]
Thanks.
maybe you could make your util more like unix sort?
for example with argument '-f' sort with ignore case, also support echo 'DCBA' | alphabet, and  the ability to read from file and write to a file would be very useful.
Regards.

**Comment by grylem** [comment]
About latest version
now
./alphabet -f cbaBAC
aAbBcC
./alphabet cbaBAC
AaBbCc

but in my opinion should be:
./alphabet cbaBAC
ABCabc
./alphabet -f cbaBAC
AaBbCc

Regards

**Comment by lille-boy** [comment]
You're right, the behavior changed. That's because I see two ways of interpreting 'ignore case':
1. Alphabetical order without caring whether a letter is upper or lower case (new solution).
2. Ascending ASCII values (first solution).

I'll revert back to have:
./alphabet cbaBAC
ABCabc
